__tests__/app.test.js

```diff
@@ -1,84 +1,123 @@
+describe("app", () => {
+  describe("GET /api/categories", () => {
+    test("200: Responds with an array of category objects", () => {
+      return request(app)
+        .get("/api/categories")
+        .expect(200)
+        .then(({ body }) => {
+          const { categories } = body;
+          expect(categories.length).toBe(4);
+          categories.forEach((category) => {
+            expect(category).toHaveProperty("slug", expect.any(String));
+            expect(category).toHaveProperty("description", expect.any(String));
+          });
         });
-
     });
+  });
 
-    
-    describe('GET /api/reviews', () => {
-        
-        it('200: responds with array of review objects', () => {
-            return request(app)
-                .get('/api/reviews')
-                .expect(200)
-                .then(({ body }) => {
-                    const { reviews } = body;
-                    expect(reviews.length).toBe(13);
-                    reviews.forEach(review => {
-                        expect(review).toHaveProperty('owner', expect.any(String));
-                        expect(review).toHaveProperty('title', expect.any(String));
-                        expect(review).toHaveProperty('review_id', expect.any(Number));
-                        expect(review).toHaveProperty('category', expect.any(String));
-                        expect(review).toHaveProperty('review_img_url', expect.any(String));
-                        expect(review).toHaveProperty('created_at', expect.any(String));
-                        expect(review).toHaveProperty('votes', expect.any(Number));
-                        expect(review).toHaveProperty('designer', expect.any(String));
-                        expect(review).toHaveProperty('comment_count', expect.any(Number));
-                    })
-                    expect(reviews[4].comment_count).toBe(3);
-            })
+  describe("GET /api/reviews", () => {
+    it("200: responds with array of review objects", () => {
+      return request(app)
+        .get("/api/reviews")
+        .expect(200)
+        .then(({ body }) => {
+          const { reviews } = body;
+          expect(reviews.length).toBe(13);
+          reviews.forEach((review) => {
+            expect(review).toHaveProperty("owner", expect.any(String));
+            expect(review).toHaveProperty("title", expect.any(String));
+            expect(review).toHaveProperty("review_id", expect.any(Number));
+            expect(review).toHaveProperty("category", expect.any(String));
+            expect(review).toHaveProperty("review_img_url", expect.any(String));
+            expect(review).toHaveProperty("created_at", expect.any(String));
+            expect(review).toHaveProperty("votes", expect.any(Number));
+            expect(review).toHaveProperty("designer", expect.any(String));
+            expect(review).toHaveProperty("comment_count", expect.any(Number));
+          });
+          expect(reviews[4].comment_count).toBe(3);
```

Comment:
How come you're checking this specific review for 3 comments?

Reply:
Hannah suggested optionally adding a test on one particular object to check the comment count is correct and as expected, in order to check the SQL worked correctly